dockers/docker-lldp/lldpd.conf.j2

```diff
@@ -1,12 +1,19 @@
+{% set mgmt_if = {} %}
 {% if MGMT_INTERFACE %}
+{% for interface in MGMT_INTERFACE.keys()|list %}
+{% if interface[1]|ipv4 %}
```

Comment:
>ipv4 [](start = 19, length = 4)

Could you add a test case similar to [https://github.com/Azure/sonic-buildimage/blob/781188f54941a2eb9e4a23a96f05986ec51ff106/src/sonic-config-engine/tests/test_j2files.py#L69](https://github.com/Azure/sonic-buildimage/blob/781188f54941a2eb9e4a23a96f05986ec51ff106/src/sonic-config-engine/tests/test_j2files.py#L69)? So it will capture regression like this.

Reply:
The test was added, please review

Reply:
Thanks for adding test! I think another useful test case is 1 ipv4 + 1 ipv6 mgmt addresses, which are common to in normal use case.

---
In reply to: [515413789](https://github.com/Azure/sonic-buildimage/pull/5699#discussion_r515413789) [](ancestors = 515413789)

Reply:
I think the test must verify  2 cases:

- IPv4 mgmt interface exists
- IPv4 mgmt interface doesn't  exist

But now I add 3 cases, when:
- IPv4 mgmt interface only exists
- IPv6 mgmt interface only exists
- IPv4 + IPv6 mgmt interface exists

Please review